build.cake

```diff
@@ -15,6 +15,9 @@ var certificatePassword = Argument("CertPassword", "");
 var mygetApiKey = Argument("MyGetApiKey", "");
 var ApplicationName = Argument("ApplicationName","ConsumerVPN");
 var UsesEVCert = Argument<bool>("EVCert", false);
+var certSubject = ""; //Add your own cert subject
+var TimeServer = "http://timestamp.sectigo.com";
```

Comment:
Suggest to name the variable as "TimeServerUrl"

Reply:
Updated to TimeServerUrl